pkg/bindinfo/global_handle.go

```diff
@@ -165,6 +170,33 @@ func (h *globalBindingHandle) setCache(c *bindCache) {
 	h.bindingCache.Store(c)
 }
 
+func (h *globalBindingHandle) getFuzzyDigestMap() map[string][]string {
+	return h.fuzzyDigestMap.Load().(map[string][]string)
+}
+
+func (h *globalBindingHandle) setFuzzyDigestMap(m map[string][]string) {
+	h.fuzzyDigestMap.Store(m)
+}
+
+func buildFuzzyDigestMap(bindRecords []*BindRecord) map[string][]string {
+	m := make(map[string][]string)
+	p := parser.New()
+	for _, bindRecord := range bindRecords {
+		for _, binding := range bindRecord.Bindings {
+			stmt, err := p.ParseOneStmt(binding.BindSQL, binding.Charset, binding.Collation)
+			if err != nil {
+				logutil.BgLogger().Warn("parse bindSQL failed", zap.String("bindSQL", binding.BindSQL), zap.Error(err))
+				p = parser.New()
```

Comment:
why create a new parser？

Reply:
Just for safety. Because there is an error, I'm afraid that the parser's internal state is wrong like some flags are not reset correctly. So create a new one for safety. @AilinKid